spec/requests/api/subscriptions/update_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'POST /api/subscriptions', type: :request do
+  let(:user) { create(:user) }
+  let(:credentials) { user.create_new_auth_token }
+
+  subject { response }
+
+  describe 'as a authenticated user, when subscribing' do
+    before do
+      post '/api/subscriptions', params: { user: { id: user.id } }, headers: credentials
+    end
+
+    it { is_expected.to have_http_status 201 }
+
+    it 'is expected to return a message that subscription status was changed' do
+      expect(response_json['message']).to eq 'You are now subscribed.'
+    end
+
+    it 'is expected that the user subscription status is changed' do
+      expect(User.last.subscribed).to eq true
+    end
+  end
+
+  describe 'as an unauthenticated user, when subsribing' do
+    before do
+      post '/api/subscriptions', params: { user: { id: user.id } }, headers: nil
```

Comment:
See commet above about passing in use info in params

Reply:
Removed